### PR TITLE
DDF-2659 Added Jarsigner functionality to catalog-transformer-zip

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR leverages Ant to sign Jars / Zip files in the ZipCompression functionality.  It also removes some obsolete binaries that were formally used to call JarSigner.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@glenhein @rzwiefel 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Security](https://github.com/orgs/codice/teams/security)
@ryeats 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@stustison
#### How should this be tested? (List steps with links to updated documentation)
Build / Install / Ingest Some Data /  `catalog:dump --include-content` and verify the jar is signed / `catalog:ingest --include-content` and verify that the signed jar passes validation.
#### What are the relevant tickets?
[DDF-2659](https://codice.atlassian.net/browse/DDF-2659)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [X] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/JarSigner.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import java.io.File;
+
+import org.apache.tools.ant.BuildException;
+import org.apache.tools.ant.Project;
+import org.apache.tools.ant.Target;
+import org.apache.tools.ant.taskdefs.SignJar;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JarSigner extends SignJar {
```
Comment (glenhein) [inline catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/JarSigner.java]: maybe some javadoc?

**Comment by jlcsmith** [comment]
Hero'd successfully.  Verified signed jar imports successfully into a DDF if it is signed by a trusted cert.  Also verified that the ingest failed if not signed by a trusted cert.

File: catalog/transformer/catalog-transformer-zip/pom.xml [hunk 2]
```diff
@@ -65,11 +75,27 @@
...
                         <Embed-Dependency>
+                            ant,
+                            ant-launcher,
                             catalog-core-api-impl,
                             platform-util,
                             zip4j
                         </Embed-Dependency>
-
+                        <Import-Package>
+                            !com.sun.tools.apt,
```
Comment (stustison) [inline catalog/transformer/catalog-transformer-zip/pom.xml]: Needing to stop things from being imported is generally a smell. It seems like these would be better served as excluded dependencies, which should also keep OWASP from complaining if one of them has a CVE.

Comment (bdeining) [inline catalog/transformer/catalog-transformer-zip/pom.xml]: These ended up not being needed

File: catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/JarSigner.java (hunk 1 above)
Comment (bdeining) [inline catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/JarSigner.java]: Added some javadocs

**Pull request merged**